lib/utils/utils.dart

```diff
@@ -214,3 +214,33 @@ Future<Response> doPost(
 
   return response;
 }
+
+Future<Response> doGet(
+    {Uri url, Map<String, String> parameters, bool sslVerify = true}) async {
+
+  ArgumentError.checkNotNull(
+      sslVerify, 'Parameter [sslVerify] must not be null!');
+
+  IOClient ioClient = IOClient(HttpClient()
+    ..badCertificateCallback =
+        ((X509Certificate cert, String host, int port) => !sslVerify));
+  // TODO Make this more general!
```

Comment:
is this still required?

Reply:
The TODO? It works how it is, it is just not very reusable. But I still want to look into that in a future build.